test/spec.js

```diff
@@ -100,55 +99,18 @@ describe('mocha-lazy-bdd', function() {
           });
         });
       });
-
-      context('another nested context', function() {
```

Comment:
These tests were asserting the behavior of `this._super`, which is removed (we're not using it anywhere, and the notion of introducing references to parent contexts in tests seems like a pattern to avoid, much like native mechanisms for disabling tests).

That said, I shouldn't couple opinionated changes (e.g: API surface reduction) alongside less opinionated ones (e.g: modernization).